thali/thalireplicationmanager.js

```diff
@@ -112,6 +148,18 @@ ThaliReplicationManager.prototype.stop = function () {
   });
 };
 
+function informDeviceIdentityListeners(err, publicKeyHash) {
```

Comment:
Actually, Matt had all the external functions at the top followed by all the internal functions. So I was following that pattern. But I have reunited the functions now.